go/p2p/api/api.go

```diff
@@ -27,10 +27,28 @@ const (
 	TopicKindTx TopicKind = "tx"
 )
 
+// Status is the P2P status of a node.
+type Status struct {
+	// ID is the public key used for P2P communication.
+	ID signature.PublicKey `json:"id"`
+
+	// Addresses is a list of configured P2P addresses used when registering the node.
+	Addresses []node.Address `json:"addresses"`
```

Comment:
Can we also show the addresses of the native p2p stack (e.g. for libp2p that would be the "peer ID" and for Tendermint P2P this would be the "p2p address")? This is useful for correlating logs as those are the addresses that are logged in corresponding peers.

Reply:
Added `PeerID` to libp2p and moved Tendermint related p2p info under `consensus.p2p` section. See PR's description for the new output.

I wanted to add that earlier for the same reasons but changed my mind as users might be confused with so many ids. For example, Tendermint's p2p status now shows 3 different ids (`id`, `peer_id`, and consensus id inside `addresses`).